### PR TITLE
Updated migrate script to include moving git repos.

### PR DESCRIPTION
**Pull request by shbatm**
I made a small update to the migrate.sh script for people that have cloned the full git repo.  It will split the old repo into the new 3 and add the new tracking info.

### CODE REVIEW

**Comment by cheesemarathon** [comment]
Thanks for this. One question, line 3 pulls the current working directory, so surely for this to work, this needs to be run from the installation dir. Am I correct with this assumption? It's not a problem, I just need to update the instructions in README.md

**Comment by shbatm** [comment]
Yes, it has to be run from wherever you have the repo cloned, which depending on when you started using this project may be different than `/opt`. I didn't want to guess where people had it. 

Maybe for the README:

To migrate your existing install to the new repo:
1. If you used easy_install.sh, please run the following command:
    ```shell
    curl -s https://raw.githubusercontent.com/mariusmotea/diyHue/master/migrate.sh | sudo bash /dev/stdin
    ```
2. If you used git to clone the repo, pull the latest version of this repo, navigate to your repo directory, and run:
    ```shell
    chmod +x migrate.sh
   ./migrate.sh
    ```
3. If you used Docker, please just pull the latest image.

**Comment by shbatm** [comment]
I would also suggest that you replace the easy_install.sh script in this repo with the one you updated in the new repo. That way if people do run the script from here (or run it from the old repo if they have it cloned) it will update everything from the new repo.

File: migrate.sh [hunk 1]
```diff
@@ -1,6 +1,24 @@
+#!/bin/bash
+
+OLDPWD=$(pwd);
+
 echo -e "\033[36m Migrating Existing diyHue Install\033[0m"
 cd /opt/hue-emulator
```
Comment (cheesemarathon) [inline migrate.sh]: Remove

File: migrate.sh [hunk 2]
```diff
@@ -1,6 +1,24 @@
+#!/bin/bash
+
+OLDPWD=$(pwd);
```
Comment (cheesemarathon) [inline migrate.sh]: Remove

File: migrate.sh [hunk 3]
```diff
@@ -1,6 +1,24 @@
...
 echo -e "\033[36m Migrating Existing diyHue Install\033[0m"
 cd /opt/hue-emulator
 OLD="http://raw.githubusercontent.com/mariusmotea/diyHue/master/BridgeEmulator/updater"
 NEW="http://raw.githubusercontent.com/diyhue/diyHue/master/BridgeEmulator/updater"
 sed -i "s#$OLD#$NEW#g" "HueEmulator3.py"
 echo -e "\033[36m Existing Install Migrated Successfully\033[0m"
+
+echo -e "\033[36m Checking if git was used."
+
+cd $OLDPWD;
```
Comment (cheesemarathon) [inline migrate.sh]: Remove

File: migrate.sh [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
+if [[ -d "./.git" ]]; then
+    echo -e "\033[36m Git Found, Updating\033[0m"
+    git remote set-url origin https://github.com/diyhue/diyHue.git
+    rm -rf ./Lights/
+    git clone https://github.com/diyhue/Lights.git
+    rm -rf ./Sensors/
+    git clone https://github.com/diyhue/Devices.git
+    cd $OLDPWD;
```
Comment (cheesemarathon) [inline migrate.sh]: Remove

**Pull request merged**